react/Header/UserAccount/UserAccount.less

```diff
@@ -84,9 +131,22 @@
   transition: transform .3s ease;
   transform-origin: 50% 50%;
   pointer-events: none;
+  .smallDeviceOnly({
+    text-align: center;
+    width: 20px;
```

Comment:
I assume this is a magic number because while it's coincidentally the same as `grid-gutter-width` it doesn't have any real relation to the grid dimensions?